packages/tsconfig/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@commercetools-frontend/tsconfig",
+  "version": "0.1.0",
+  "description": "Shared TypeScript config for Custom Applications",
+  "bugs": "https://github.com/commercetools/merchant-center-application-kit/issues",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/commercetools/merchant-center-application-kit.git",
+    "directory": "packages/tsconfig"
+  },
+  "homepage": "https://docs.commercetools.com/custom-applications",
+  "keywords": [
+    "config",
+    "configuration",
+    "javascript",
+    "frontend",
+    "toolkit",
+    "tsconfig",
+    "typescript",
+    "ts"
+  ],
+  "license": "MIT",
+  "publishConfig": {
+    "access": "public"
+  },
+  "engines": {
+    "node": ">=14"
```

Comment:
Good point, but it seems to me it’s not consistent across the monorepo. 
`create-mc-app` is the only package where `v15` is explicitly excluded.

Reply:
Yes, we changed in that package when we noticed the problem over there.

I guess it makes sense for the new packages to follow this rule to avoid possible problems for our users.